Mathlib/Combinatorics/SetFamily/Shadow.lean

```diff
@@ -104,13 +107,52 @@ theorem mem_shadow_iff_insert_mem : s ∈ ∂ 𝒜 ↔ ∃ (a : _) (_ : a ∉ s)
     exact ⟨insert a s, hs, a, mem_insert_self _ _, erase_insert ha⟩
 #align finset.mem_shadow_iff_insert_mem Finset.mem_shadow_iff_insert_mem
 
+/-- `t ∈ ∂𝒜` iff `t` is exactly one element less than something from `𝒜` -/
```

Comment:
Perhaps a docstring referencing the very similar lemma `mem_shadow_iff_exists_mem_card_add_one` below would be nice (and similar for iterated). I expect one can be used to golf the other?